.github/workflows/kserve_m2m_test.yaml

```diff
@@ -44,11 +43,46 @@ jobs:
     - name: Install knative
       run: ./tests/gh-actions/install_knative.sh
 
-    - name: Build & Apply manifests
+    - name: Install KServe
       run: ./tests/gh-actions/install_kserve.sh
 
-    - name: Create test namespace
-      run: kubectl create ns kserve-test
+    - name: Install KF Multi Tenancy
+      run: ./tests/gh-actions/install_multi_tenancy.sh
+
+    - name: Install kubeflow-istio-resources
+      run: kustomize build common/istio-1-24/kubeflow-istio-resources/base | kubectl apply -f -
```

Comment:
also this

```
    - name: Install kubeflow-istio-resources
      run: kustomize build common/istio-1-24/kubeflow-istio-resources/base | kubectl apply -f -
```
should be higher up as in https://github.com/kubeflow/manifests/blob/88801f6ce84ac7eab3e2883dd3d80cb63ccb6ab3/.github/workflows/pipeline_run_from_notebook.yaml#L39